lib/pa_ess/utilities.ex

```diff
@@ -552,7 +552,7 @@ defmodule PaEss.Utilities do
   @spec prediction_approaching?(Predictions.Prediction.t(), boolean()) :: boolean()
   def prediction_approaching?(prediction, terminal?) do
     !terminal? and !prediction.stopped_at_predicted_stop? and
-      prediction_seconds(prediction, terminal?) in 31..45
+      prediction_seconds(prediction, terminal?) in 1..45
```

Comment:
Predictions can go all the way down to 0, so let's make this `0..45` just to avoid any surprise edge cases in the future.